README.md

```diff
@@ -220,106 +216,15 @@ autoscaling policies.
 * **`min_capacity`** - (Required) Minimum task count for autoscaling (this will also be used to define the initial
   desired count of the ECS Fargate Service)
 * **`max_capacity`** - (Required) Maximum task count for autoscaling
+* **`cpu_percentage_target`** (Optional) What the desired average CPU percentage is
```

Comment:
(nit) should we share the default here?
```suggestion
* **`cpu_percentage_target`** (Optional) What the desired average CPU percentage is. Defaults to `75`
```

Reply:
For now I have made it a required field.